Packs/IBMResilientSystems/Scripts/IbmUpdateNote/IbmUpdateNote.py

```diff
@@ -0,0 +1,32 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def update_note(args: Dict[str, Any]) -> CommandResults:
+    note_id = args.get('note_id', '')
+    note_body = args.get('note_body', '')
+    tags = argToList(args.get('tags', ''))
+    remote_incident_id = demisto.incident()['dbotMirrorId']
+    demisto.debug(f'update_note {note_body=} | {remote_incident_id}')
```

Comment:
is remote_incident_id missing a = as well? like note_body?

Reply:
yes, thanks